invesalius/data/styles.py

```diff
@@ -2718,8 +2718,8 @@ def __init__(self):
         self.con_2d = 4
         self.con_3d = 6
 
-        self.t0 = None
-        self.t1 = None
+        self.t0: "int | None" = None
+        self.t1: "int | None" = None
```

Comment:
It's not better to use Optional[int]?

Reply:
I wasn't type hinting this file so I wanted to make minimal changes to it. In practice adding newer typing features is possible by making type hints explicitly strings. For now I think it is better to just use the standard for python 3.8 and upgrade later.